app/src/main/java/org/systers/mentorship/view/adapters/TasksAdapter.kt

```diff
@@ -1,37 +1,58 @@
 package org.systers.mentorship.view.adapters
 
-import androidx.recyclerview.widget.RecyclerView
+import android.content.Context
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.appcompat.app.AlertDialog
+import androidx.recyclerview.widget.RecyclerView
 import kotlinx.android.synthetic.main.task_list_item.view.*
-import org.systers.mentorship.MentorshipApplication
 import org.systers.mentorship.R
 import org.systers.mentorship.models.Task
 
 /**
  * This class represents the adapter that fills in each view of the Tasks recyclerView
- * @param taskstsList list of tasks taken up by the mentee
- * @param markTask function to be called when an item from Tasks list is clicked
+ * @param tasksList list of tasks taken up by the mentee
+ * @param checkFunc function to be called when an item from Tasks list is clicked
  */
```

Comment:
Just one more change. Sorry I forgot about this. Change **checkFunc** to **markTask** in this doc string

Reply:
Sure.

Reply:
Done.